app/InternetAccess.php

```diff
@@ -8,7 +8,7 @@ class InternetAccess extends Model
 {
     protected $primaryKey = 'user_id';
 
-    protected $fillable = ['user_id'];
+    protected $fillable = ['user_id', 'wifi_username'];
     protected $hidden = ['wifi_password'];
```

Comment:
Are we storing passwords here?
You know that I'm not really a fan of that.

Reply:
These are autogenerated random passwords only for the wifi, so it's okay I think.

Reply:
No passwords should be stored as plain text. Regardless of its purpose.